starter/src/styles/styles.pcss

```diff
@@ -1,9 +1,3 @@
 /*
 IMPORTANT: Stark styles are provided as PostCSS styles so they should be imported in a PCSS file!
 */
-
-/* Stark variables */
-@import "../../node_modules/@nationalbankbelgium/stark-ui/assets/styles/_variables.pcss";
```

Comment:
Same as above, without this I guess the app-logo component won't be displayed correctly 😕